README.md

```diff
@@ -480,6 +492,7 @@ h
 * [Google Web Toolkit](http://www.gwtproject.org/) - Toolbox which includes a Java-to-JavaScript compiler for client-side code, XML parser, API for RPC, JUnit integration, internationalization support and widgets for the GUI.
 * [Grails](https://grails.org/) - Groovy framework with the aim to provide a highly productive environment by favoring convention over configuration, no XML and support for mixins.
 * [Ninja](http://www.ninjaframework.org/) - Full stack web framework for Java. Rock solid, fast and super productive.
+* [Oracle ADF](http://www.oracle.com/technetwork/developer-tools/adf/overview/index.html) - Providers a free and a commercial version framework to write web and mobile applications, following an event-driven approach.
```

Comment:
JDeveloper comes prepackaged with chock full of modules and plugins for working with Oracle specific technologies (i.e. ADF). It's really an enterprise-style development environment that I would also argue is not appropriate for this list. 

Equally, I have never heard anyone using ADF for a personal project, nor would I recommend anyone to use it on any kind of non-enterprise project, so to compare it with PrimeFaces is a bit disingenuous.